[Feat 9] agregar contributions y actualizar documentación de instalación (#10)
taecontrol/larastats-docs: docs/installation.md

## Installation via Composer

```bash
composer require taecontrol/moonguard
```

> ## Manual Installation

Reviewer: ```suggestion
## Local Development
```

Clone the [MoonGuard Repository](https://github.com/teacontrol/moonguard)

> Once you have clone it, in your `composer.json` of your project add the

Reviewer: ```suggestion
Once you have clone it, update the `composer.json` of your project: 
```

**repository** key:

```json
{
    "repositories": [
        {
            "type": "path",
            "url": "../<my-package-path>/moonguard"
        }
    ]
}
```

> Then in the **require** key add the MoonGuard package:

Reviewer: ```suggestion
Then add MoonGuard package in the `require` key:
```

```json
{
    "require":  {
        "taecontrol/moonguard": "dev-main"
    }
}
```
Finally you can run the `composer update` command to install MoonGuard to your project.

```bash
composer update
```

> At this point you should have to register two service providers in

Reviewer: ```suggestion
At this point you have to register two service providers in
```
